### ISSUE TITLE
Integrate Power-analysis with similarity

### ISSUE THREAD

**Issue by JohnHadish** [SystemsGenetics/KINC#144]
The power analysis step should be integrated with the similarity step as an additional parameter. This will reduce the number of steps a user must take to produce a network.

**Comment by spficklin** [comment]
Yeah, I agree.  The fewer steps the less mistakes can be made.  But this would be a major change to the workflow and should result in a major  release change (e.g. 3.5)

**Comment by bentsherman** [comment]
I am 100% in favor of doing this, as it reduces a redundant read and write of ccm/cmx files.
